packages/block-editor/src/components/rich-text/index.js

```diff
@@ -509,13 +520,27 @@ const RichTextContainer = compose( [
 			isSelected = selectionStart.clientId === clientId;
 		}
 
+		let extraProps = {}
+		if ( Platform.OS !== 'web' ) {
```

Comment:
These two props don't seem to conflict with the web stuff, so maybe we can skip the check and always inject them.  Maybe we could have something like a `__native` prefix to indicate better that they are for native-specific behavior.

Reply:
Will the check for platform not make it obvious enough? I could change the code to `Platform.OS == `native` now that we have our Platform component in.